main.lua

```diff
@@ -15,71 +15,134 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 --]]
-
-local bundle = require('luvi').bundle
+-- Convoluted boostrapping?
+local bundle = require("luvi").bundle
 loadstring(bundle.readfile("luvit-loader.lua"), "bundle:luvit-loader.lua")()
 
-local uv = require('uv')
-require('snapshot')
+-- Upvalues
+local uv = require("uv")
+local version = require("./package").version
+local log = require("log").log
+require("snapshot")
+
+-- Global setup
+_G.p = require("pretty-print").prettyPrint -- TBD: Can't this be removed? Luvi already injects the same global AFAIK
+
+-- Settings
+local EXIT_SUCCESS = 0
+local EXIT_FAILURE = -1
+
 local aliases = {
-  ["-v"] = "version",
-  ["-h"] = "help",
+	["-v"] = "version",
+	["-h"] = "help"
 }
 
-local function exit(status)
-  uv.walk(function(handle)
-    if handle then
-      local function close()
-        if not handle:is_closing() then handle:close() end
-      end
-      if handle.shutdown then
-        handle:shutdown(close)
-      else
-        close()
-      end
-    end
-  end)
-  uv.run()
-  os.exit(status)
+local Lit = {}
```

Comment:
What's this for?

Reply:
Good question! I think I planned to move the ``CLI`` container elsewhere and use it as a namespace in a top-level ``Lit`` module, but that seems too ambitious for this PR so I can probably remove it (for now).